code/game/objects/items/weapons/grenades/frag.dm

```diff
@@ -6,8 +6,9 @@
 	loadable = TRUE
 
 	var/fragment_type = /obj/item/projectile/bullet/pellet/fragment/strong
-	var/num_fragments = 150  //total number of fragments produced by the grenade
-	var/fragment_damage = 5
+	var/num_fragments = 50  //total number of fragments produced by the grenade
+	var/fragment_damage = 15
+	var/fragment_penetration = 15
```

Comment:
Where this variable is used?

Reply:
I don't understand which variable you mean, most of these were already here except for fragment_penetration.

Reply:
I'm indeed talking about it. It isn't used anywhere. Did you test your PR? It will cause an error or atleast a warning when compiling.